sympy/polys/rootisolation.py

```diff
@@ -1792,9 +1792,10 @@ class ComplexInterval(object):
     Examples
     ========
 
-    >>> from sympy import RootOf, Rational, S
+    >>> from sympy import CRootOf, Rational, S
     >>> from sympy.abc import x
-    >>> root = RootOf(x**10 - 2*x + 3, 9)
+    >>> CRootOf.clear_cache()
```

Comment:
I agree. It's annoying to have to put test specific stuff in a doctest, because they are supposed to serve as examples for users.

Reply:
> I agree

I added an inline comment to the docstrings and a docstring to the function. If this is OK, this can be committed when tests are done and @jksuom approves.